### DISCUSSION TITLE
Youtube Transcript/Video Chapters

### DISCUSSION THREAD

**Discussion by KijanaG** [jdepoix/youtube-transcript-api#186] [Ideas]
Is there a way to recognize the chapters of each section of the transcript?

**Comment by jdepoix** [comment]
Hi @KijanaG,
there recently was a discussion around adding chapter support in #182. I guess the TLDR is "it doesn't really fit the scope of this module", but it looks like you could use youtube-dl to find out which chapter each snippet of your transcript belongs to.

**Reply by KijanaG** [reply]
Hey thanks for the reply!! I'll look into it.
